helmfile/charts/networkpolicy-service/templates/ingress-nginx/ingress-nginx.yaml

```
@@ -0,0 +1,189 @@
+              app.kubernetes.io/instance: kube-prometheus-stack-prometheus
+      ports:
+        - port: 10254
+    - from:
+        - namespaceSelector:
+            matchExpressions:
+              - key: kubernetes.io/metadata.name
+                operator: In
+                values:
+                  - monitoring
+        - podSelector:
+            matchLabels:
+              app.kubernetes.io/instance: kube-prometheus-stack
+      ports:
+        - port: 10250
+    - from:
+{{- range $IP := .Values.global.scApiserver.ips }}
+      - ipBlock:
+          cidr: {{ $IP }}
+{{- end }}
+      ports:
+        - protocol: TCP
+          port: 8443 ## admission
+  egress:
+    - ports:
+      - port: 53
+        protocol: UDP
+    - to:
+{{- if .Values.ingressNginx.ingressOverride.enabled }}
+  {{- range $IP := .Values.ingressNginx.ingressOverride.ips }}
+      - ipBlock:
+          cidr: {{ $IP }}
+  {{- end }}
+{{- else }}
+  {{- range $IP := .Values.global.scIngress.ips }}
+      - ipBlock:
+          cidr: {{ $IP }}
+  {{- end }}
+{{- end }}
+      ports:
+        - protocol: TCP
+          port: 443
+        - protocol: TCP
+          port: 80
+    - to:
+        {{- range $IP := .Values.global.scApiserver.ips }}
+        - ipBlock:
+            cidr: {{ $IP }}
+        {{- end }}
+      ports:
+        - protocol: TCP
+          port: {{ .Values.global.scApiserver.port }}
+    - to:
+        - namespaceSelector:
+            matchExpressions:
+              - key: kubernetes.io/metadata.name
+                operator: In
+                values:
+                  - monitoring
+          podSelector:
+            matchLabels:
+              app.kubernetes.io/name: grafana
+      ports:
+        - port: 3000
+    - to:
+        - namespaceSelector:
+            matchExpressions:
+              - key: kubernetes.io/metadata.name
+                operator: In
+                values:
+                  - opensearch-system
+          podSelector:
+            matchLabels:
+              app: opensearch-dashboards
+      ports:
+        - port: 5601
+    - to:
+        - namespaceSelector:
+            matchExpressions:
+              - key: kubernetes.io/metadata.name
+                operator: In
+                values:
+                  - opensearch-system
+          podSelector:
+            matchLabels:
+              app.kubernetes.io/component: opensearch-master
+      ports:
+        - port: 9200
+    - to:
+        - namespaceSelector:
+            matchExpressions:
+              - key: kubernetes.io/metadata.name
+                operator: In
+                values:
+                  - harbor
+          podSelector:
+            matchLabels:
+              component: core
+      ports:
+        - port: 8080
+    - to:
+        - namespaceSelector:
+            matchExpressions:
+              - key: kubernetes.io/metadata.name
+                operator: In
+                values:
+                  - harbor
```

You can change all these namespace selectore to be
```suggestion
            matchLabels:
              kubernetes.io/metadata.name: harbor
```